config/config.go

```diff
@@ -887,6 +887,14 @@ func (c *Config) ToOptions() skipper.Options {
 
 		ClusterRatelimitMaxGroupShards: c.ClusterRatelimitMaxGroupShards,
 	}
+	for _, rcci := range *c.CloneRoute {
```

Comment:
Why do we need to deference pointer here?
Since `routeChangerConfig` became slice I think `c.CloneRoute` type should not be pointer type anymore. See how `multiFlag` is used

Reply:
Yes in Go you basically never need to dereference ptr yourself.